src/Sulu/Bundle/MarkupBundle/Markup/HtmlMarkupParser.php

```diff
@@ -19,8 +19,9 @@
 class HtmlMarkupParser implements MarkupParserInterface
 {
     const ATTRIBUTE_REGEX = '/(?<name>\b\w+\b)\s*=\s*"(?<value>[^"]*)"/';
-    const CONTENT_REGEX = '/(?:>(?<content>.*)<)/';
-    const TAG_REGEX = '/(?<tag><%s:(?<name>[a-z]*)[^\/>]*(?:\/>|>.*<\/%s:[^\/>]*>))/';
```

Comment:
this change avoid the match on multiple tags which could lead into use the start tag of one and the end tag of another one.